docs/developers/xmex-transfer.md

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 3
+id: xmex-transfer
+---
+
+[comment]: # (mx-exclude-context)
+
+# XMEX transferability in community projects
+
+Projects developing on top of xExchange smart contracts might need the ability to work with XMEX, either acting as proxy to transfer it from/to the xExchange yield generating contracts, or by accepting it as means of payment for specific business cases. Many of the core principles of XMEX are designed around its account binding property, which in this case removed the open transferability of XMEX. For this reason, we have designed an XMEX transfer mechanisms for projects to use, in order to assure and streamline the conservation of the XMEX benefits and properties without too much development effort required on implementing project's side.
+
+[comment]: # (mx-context-auto)
+
+## Transfer method and specifications
+
+On each xExchange environment we have deployed an [XMEX wrapping contract](https://github.com/multiversx/mx-exchange-sc/tree/main/locked-asset/locked-token-wrapper) to facilitate the transferability for implementing projects. The XMEX wrapping contract accepts XMEX, removes its energy and properties from the sender and gives back WXMEX. The WXMEX token does not hold energy or benefits in its wrapped form, apart from it's ability to be transfered if the holder address is whitelisted. 
```

Comment:
to facilitate the transferability for projects building on top of xMEX

Reply:
done.